### PR TITLE
[AD-763] [JDBC] remove table null check in getPrimaryKeys and getImportedKeys

### PR DESCRIPTION
**Pull request by birschick-bq**
### Summary

[AD-763] [JDBC] remove table null check in getPrimaryKeys and getImportedKeys

### Description

Ensure table name is explicit and no longer accept a null value for table name.

### Related Issue

https://bitquill.atlassian.net/browse/AD-763

### Additional Reviewers
@affonsoBQ
@alinaliBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,4 +8,4 @@ APPLICATION_NAME=Amazon DocumentDB JDBC Driver
 # Comment out if not creating a beta version
 #BETA_VERSION=1
 # Uncomment out next line to disable remote integration tests
-#runRemoteIntegrationTests=false
+runRemoteIntegrationTests=false
```
Comment (alinaliBQ) [inline gradle.properties]: I think remote tests should still be run on GitHub?

**Review by andiemontoyeah — APPROVED** [review]
LGTM after reverting gradle.properties

File: gradle.properties (hunk 1 above)
Comment (birschick-bq) [inline gradle.properties]:
```suggestion
#runRemoteIntegrationTests=false
```

Comment (birschick-bq) [reply gradle.properties]: Oops. Thanks.

**Review by alinaliBQ — APPROVED** [review]
lgtm!

File: src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java [hunk 2]
```diff
@@ -495,7 +495,7 @@ public ResultSet getPrimaryKeys(final String catalog, final String schema,
         final List<List<Object>> metaData = new ArrayList<>();
         if (schema == null || properties.getDatabase().equals(schema)) {
             for (String tableName : databaseMetadata.getTableSchemaMap().keySet()) {
-                if (table == null || tableName.equals(table)) {
+                if (tableName.equals(table)) {
                     final DocumentDbSchemaTable metadataTable = databaseMetadata
```
Comment (RoyZhang2022) [inline src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java:499]: Should this be changed to tableName.matches(regexTablePattern)?

Comment (alinaliBQ) [reply src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java:499]: I don't think so. According to the documentation, for getPrimaryKeys, the tableName should match as it is stored in the database. [reference to oracle documentation](https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getPrimaryKeys(java.lang.String,%20java.lang.String,%20java.lang.String))

Comment (RoyZhang2022) [reply src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java:499]: Ok, I see. The table name can not be a search pattern.

**Pull request merged**